test/test_primitives/assert_helper.h

```diff
@@ -29,6 +29,37 @@ inline void assert_bounding_box(const Collection<T> &collection, const unsigned
   CHECK(bmax[2] == Approx(true_max[2]));
 }
 
+/**
+    If multiple IDs (more objects) are used, all are checked and a global minimum and maximum are searched for
+    @param num_prim_id : Number of prim_ids in object
+*/
+template<typename T, template<typename> typename Collection>
+inline void assert_bounding_box_multi_prim_id(const Collection<T> &collection, const unsigned int prim_id,
```

Comment:
`multi_prim_id` is exactly what a collection is, so a better name would probably be `assert_bounding_box_collection` .

Furthermore, prim_id shouldn't be needed in this case, because it does not test a single primitive.

Reply:
renamed and removed "num_prim_id"